M/MPIR/build_tarballs.jl

```diff
@@ -34,7 +34,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = Dependency[
+    Dependency("YASM_jll"),
```

Comment:
This is a _**build**_ tool
```suggestion
dependencies = [
    HostBuildDependency("YASM_jll"),
```